keyboards/monsgeek/m2/readme.md

```diff
@@ -0,0 +1,25 @@
+# MonsGeek M2
+[![M2](https://s21.ax1x.com/2024/03/23/pFhDAqH.md.png)](https://imgse.com/i/pFhDAqH)
```

Comment:
Why is the image a click through to an unrelated site?
And why is it hosted on an unknown link,'s21.az1x'—what ` - what is the guarantee of permanence on this? *(The PR guidelines is to host on `imgur`.)*